camkes/parser/tests/test_rpi4.dts

```diff
@@ -0,0 +1,376 @@
+/*
+ * SPDX-License-Identifier: GPL-2.0-only
+ * Copyright Linux
+ */
```

Comment:
For test_rpi4.dtb I believe the tool just requires a new file `test_rpi4.dtb.license` with the contents the same as the 4 lines of this file.